unit_tests/engine/geometry_string.cpp

```diff
@@ -22,14 +25,11 @@ BOOST_AUTO_TEST_CASE(decode)
 
     // Test coordinates; these would be the coordinates we give the loc parameter,
     // e.g. loc=10.00,10.0&loc=10.01,10.1...
-    util::Coordinate coord1(util::FloatLongitude{10.0}, util::FloatLatitude{10.00});
-    util::Coordinate coord2(util::FloatLongitude{10.1}, util::FloatLatitude{10.01});
-    util::Coordinate coord3(util::FloatLongitude{10.2}, util::FloatLatitude{10.02});
-    util::Coordinate coord4(util::FloatLongitude{10.3}, util::FloatLatitude{10.03});
-    util::Coordinate coord5(util::FloatLongitude{10.4}, util::FloatLatitude{10.04});
-
-    // Put the test coordinates into the vector for comparison
-    std::vector<util::Coordinate> cmp_coords = {coord1, coord2, coord3, coord4, coord5};
+    std::vector<util::Coordinate> cmp_coords = {{Lng{10.0}, Lat{10.00}},
+                                                {Lng{10.1}, Lat{10.01}},
+                                                {Lng{10.2}, Lat{10.02}},
+                                                {Lng{10.3}, Lat{10.03}},
+                                                {Lng{10.4}, Lat{10.04}}};
```

Comment:
@TheMarex I believe you pointed out that you feel negatively about having abbreviations in the code. Would this fall into the definition, or would it be alright in the unit tests here? I would be fine with having them here.